perk-store-services/src/main/java/org/exoplatform/perkstore/dao/PerkStoreOrderDAO.java

```diff
@@ -249,7 +249,17 @@ private StringBuilder getOrderFilterQueryString(String username, OrderFilter fil
         } else {
           firstConditionAdded = true;
         }
-        if (filter.getProductId() ==0 && filter.isMyOrders()) {
+        if (filter.isAdministratorOrders()) {
```

Comment:
Could you use the same name for the property as in the PR title ?
```suggestion
        if (filter.isStoreManager()) {
```